package.json

```diff
@@ -41,10 +41,11 @@
     "ember-cli-jshint": "^1.0.0",
     "ember-cli-qunit": "^2.0.0",
     "ember-cli-release": "^0.2.9",
-    "ember-cli-rollbar": "0.3.1 ",
+    "ember-rollbar-client": "0.2.2",
     "ember-cli-sass": "^6.1.1",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^1.2.0",
+    "ember-rollbar-client": "0.2.2",
```

Comment:
@viveksriram1 , Why this package is added twice?

Reply:
Removing it .